submissions/LuckyDnepr/memory-pair-game/style.css

```diff
@@ -0,0 +1,421 @@
+@font-face {
+    font-family: "Mono";
+    src: url(./assets/SpaceMono-Regular.ttf);
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: "Mono", "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+    background-color: #ffeee4;
+}
+
+.fader {
+    height: 100vh;
+    width: 100vw;
+    position: absolute;
+    background-color: rgba(0, 0, 0, 0.733);
+    z-index: 1000;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    width: 60%;
+    margin: auto;
+    height: calc(100vh - 4px);
+    /* border: 1px solid green; */
+}
+
+.header {
+    height: 3em;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.title {
+    text-shadow: 3px 3px 5px steelblue;
+}
+
+.main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: calc(100% - 5em);
+}
+
+.nav {
+    height: 100%;
+    display: grid;
+    grid-template-columns: 2fr 3fr;
+    grid-template-rows: 20% auto 20%;
+    grid-template-areas:
+        "fieldsize fieldsize"
+        "themes themeImage"
+        "startbtn startbtn";
+}
+
+.nav_menu {
+    display: flex;
+    border: 1px solid steelblue;
+    border-radius: 10px;
+    margin: 1em;
+}
+
+.fieldsize {
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    grid-area: fieldsize;
+}
+
+.themes {
+    flex-direction: column;
+    grid-area: themes;
+    justify-content: center;
+    align-items: center;
+}
+
+.nav_menu input[type="radio"] {
+    appearance: none;
+    position: fixed;
+    width: 0;
+}
+
+.nav_menu label {
```

Comment:
Avoid styling by HTML tags except you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

Reply:
Why did I use tag-based styling if I added classes to all these elements? The question is more for me. Practice. More practice.